mobile/src/people/employee-card-with-avatar.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { Component } from 'react';
+import { Animated, Easing, View, Image, Dimensions, StyleSheet, FlatList, ListRenderItemInfo } from 'react-native';
+import { Avatar } from './avatar';
+import { Employee } from '../reducers/organization/employee.model';
+import { StyledText } from '../override/styled-text';
+import { companyItemStyles as styles, companyTinyItemStyles as tinyStyles} from './styles';
+
+interface EmployeeCardWithAvatarProps {
+    employee?: Employee;
+    departmentAbbreviation?: string;
+    chiefId?: string;
+    employees?: Employee[];
+    leftNeighbor?: Employee;
+    rightNeighbor?: Employee;
+    treeLevel?: number;
+    stretchToFitScreen?: boolean;
+}
+
+export class EmployeeCardWithAvatar extends Component<EmployeeCardWithAvatarProps> {
+    public state = {
+        fadeInAnim: new Animated.Value(0),
+        fadeOutAnim: new Animated.Value(1),
+        isNeighboursAvatarsVisible: true
+    };
+
+    public revealNeighboursAvatars(visibility: boolean) {
+        this.setState({isNeighboursAvatarsVisible: visibility});
+    }
+
+    public componentDidMount() {
+        Animated.timing(
+          this.state.fadeInAnim,
+          {
+            toValue: 0.7,
+            duration: 1000,
+          }
+        ).start();
+
+        Animated.timing(
+            this.state.fadeOutAnim,
+            {
+              toValue: 0,
+              duration: 500,
+            }
+        ).start();
+    }
+
+    public render() {
+        const employee = this.props.employee;
```

Comment:
100 lines of code in this method. Split it please

Reply:
addressed. highly likely it will be split into few components as animation tune up follow.